core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -1143,3 +1141,5 @@ class ZkAdminManager(val config: KafkaConfig,
     retval
   }
 }
+
+private[server] final case class RequestStatus(user: String, mechanism: Option[ScramMechanism], legalRequest: Boolean, iterations: Int)
```

Comment:
Note that I have renamed `requestStatus` to `RequestStatus` (classes should start with a capital) and made it private since `RequestStatus` is only ever used internally.

Reply:
Nice catch. Wondering why it was creating using lower case on the first place as it doesn't follow any Scala convention.